### ISSUE TITLE
Editor does not work in Safari

### ISSUE THREAD

**Issue by moberwasserlechner** [moberwasserlechner/vaadin-medium-editor#12]
The js library supports safari and work but the wrapper does not show the toolbar.

**Comment by moberwasserlechner** [comment]
Tried this with a app based on Vaadin 7.6.1 and it works. Demo runs with Vaadin 7.4.0.

**Comment by moberwasserlechner** [comment]
Vaadin version has no impact at all.

The problem is `-webkit-user-select: none;` at the top of the css hierarchy. After removing the line in web inspector the toolbar works immediatley.

```
.v-app {
	height: 100%;
	-webkit-tap-highlight-color: transparent;
	-webkit-text-size-adjust: 100%;
	-ms-text-size-adjust: 100%;
	-webkit-text-size-adjust: 100%;
	-webkit-user-select: none;
	-moz-user-select: none;
	-ms-user-select: none;
	user-select: none;
}
```
Thought its funny that FF apparently ignores `-moz-user-select` in the latest version.

This stackoverflow entry pointed the problem out: http://stackoverflow.com/questions/4251227/contenteditable-div-not-actually-editable-in-webkit

**Possible solutions**
* Use a textarea instead of the contenteditable div, which might also fix #13.
* Set a style to the editor's div that enables user-select.
```
        -webkit-user-select: text;
	-moz-user-select: text;
	-ms-user-select: text;
	user-select: text;
```
* Let the addon users handle it in their application by setting sth like
```
*[contenteditable] {
        -webkit-user-select: text;
	-moz-user-select: text;
	-ms-user-select: text;
	user-select: text;
}
```
* Create a issue for Vaadin and/or the Valo theme to allow text selection for contenteditable divs. https://github.com/vaadin/vaadin/blob/7.6/WebContent/VAADIN/themes/valo/shared/_global.scss#58

**Conclusion**
The first 2 solutions should work
The third is just a work around and users would not be happy with it.
The fourth would be nice but the addon would therefore only work in the latest version of Vaadin.

**Comment by moberwasserlechner** [comment]
I solved the problem by setting `user-select` styles in the connector javascript. With this solution the addon does not need a css-file of its own.
